scripts/package/fpsci_packager.py

```diff
@@ -0,0 +1,68 @@
+# This script generates a packaging script based on the files used by the last successful run of the program.
+# You need to run the generated script to create the distribution directory, then zip that up to send to people.
+
+# Name of the packaging script to generate
+outputScript = 'package.sh'
+
+# Path of the log.txt file to use
+inputLog = 'data-files/log.txt'
+
+# The list of files that should always be excluded
+excludeList = ['log.txt', 'keymap.Any', 'systemconfig.Any']
+# The list of files that should be excluded for release builds, but may want to be included for specific experiment distributions
+genericExcludeList = ['experimentconfig.Any', 'startupconfig.Any', 'userconfig.Any', 'userstatus.Any']
+
+# List of paths to search for to merge into the same distribution directory
+basePath = ['data-files', 'game', 'common']
```

Comment:
In the `g3d_packager.py` script I also included `research` here, but we may not be using anything from that sub-directory.

Reply:
I don't think FPSci is using anything from the `research` directory, but it may be needed in the future. Something we should keep an eye on.